packages/web-frontend/src/historyNavigation/utils.js

```diff
@@ -86,14 +87,14 @@ export const isLocationEqual = (a, b) => {
   return true;
 };
 
-export const translateSearchToInternal = search => {
+const parseSearch = search => {
```

Comment:
I went with `parseSearch` to match `queryString.parse` rather than `decodeSearch`.

Reply:
Good call!